### PR TITLE
Fix avatar name

### LINKED ISSUE #95: Fix: AvatarName if name === '""' or length is empty return address
after filtering out whitespaces

### PR DESCRIPTION
**Pull request by yayay927**
closes #95 

## Checklist
- [x] Ran Linting
- [x] Targeted PR against correct branch.
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit tests.
- [x] Added an entry to the `CHANGELOG.md` file.
- [x] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,3 +19,11 @@
...
 - Add tokens chart ([\#76](https://github.com/forbole/big-dipper-2.0-solana/issues/76))
 - Add Stake Weight Component ([\#41](https://github.com/forbole/big-dipper-2.0-solana/issues/41))
+- Fix: update general chain config report issue url ([\#98](https://github.com/forbole/big-dipper-2.0-solana/issues/98))
+- Update: hash and messages to signature and instruction in transaction sections ([\#103](https://github.com/forbole/big-dipper-2.0-solana/issues/103))
+- Update: dockerfile and .env names ([\#89](https://github.com/forbole/big-dipper-2.0-solana/issues/89))
+- Home: Fix overflowing in latest blocks responsive views ([\#85](https://github.com/forbole/big-dipper-2.0-solana/issues/85))
+- Remove tx style option in global settings ([\#83](https://github.com/forbole/big-dipper-2.0-solana/issues/83))
+- Update 404 and not found logo ([\#99](https://github.com/forbole/big-dipper-2.0-solana/issues/99))
+- Update gql ([\#100](https://github.com/forbole/big-dipper-2.0-solana/issues/100))
+- Fix AvatarName if name === '""' or length is empty return address ([\#95](https://github.com/forbole/big-dipper-2.0-solana/issues/95))
```
Comment (ryuash) [inline CHANGELOG.md:29]: For future reference please just focus on your own issue :0. I sometimes don't add it because I don't think it's necessary

File: src/components/avatar_name/index.tsx [hunk 2]
```diff
@@ -10,14 +10,21 @@ const AvatarName: React.FC<AvatarName> = ({
...
     <Link href={href(address)}>
       <a>
         <div className={classnames(className, classes.root)}>
           <Avatar address={address} imageUrl={imageUrl} />
           <Typography variant="body1">
-            {name || address}
+            {name ? avatarName : address}
           </Typography>
         </div>
       </a>
```
Comment (ryuash) [inline src/components/avatar_name/index.tsx:30]: So if the name = "q" or "a" we will show address instead?
```
else if (name.length === 1)
```

Aside from the above comment, this works but I think we can do better. 
The title was extremely misleading and it's 100% my fault. 

Since most of the issue is coming from the validators itself so we can probably fix this without touching the `AvatarName` component at all.

Method 1:
In `src/recoil/validators/hooks.ts` we can handle it here. On line 49 `const moniker = name || '';`
We can add some more checks such as:

```
let moniker = '';
if (name.replace(/ /g, '').length && name !== '""') { // check that it has length and it does not equal `'""'`
  moniker = name;
}
```

I would prefer to do it on this level in the event that we might actually want to `'"'''` on another account. Just not validators. Setting the assignment here gives us more control on a local level as oppose to the global component level

Method 2:
Basically what you did here.

Comments:
You can decide how you want to proceed as long as you can back it up. Just want to show there may be another method to this

**Review by ryuash — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (yayay927) [reply CHANGELOG.md:29]: I see. I thought you forgot so added everything 😅 Should I remove them and only leave this issue number?

Comment (yayay927) [reply CHANGELOG.md:29]: I will remove~

File: src/components/avatar_name/index.tsx (hunk 2 above)
Comment (yayay927) [reply src/components/avatar_name/index.tsx]: Updated to method 1.

**Review by ryuash — APPROVED** [review]
